src/modules/LikeEvent.js

```diff
@@ -0,0 +1,16 @@
+import { postLikesImg } from './Involvementapi.js';
+
+function SetLikes() {
+  const LikeBtn = document.querySelectorAll('.bx-heart');
+
+  LikeBtn.forEach((element) => {
+    element.addEventListener('click', async (e) => {
+      const ArtWorkId = e.target.dataset.artworkId;
+      e.stopPropagation();
+      postLikesImg(ArtWorkId);
+      window.location.reload();
```

Comment:
This line reloads the page. Although this is acceptable, I think we can use JavaScript to reload the artwork cards. That way, our app will be a **single-page application** that uses JavaScript to update only some parts of the page. Since this avoids reloading the page, it will result in a better user experience. However, this is only a suggestion, and you don't have to fix it for this PR to get approved.